pymoveit2/moveit2.py

```diff
@@ -872,6 +894,52 @@ def _plan_kinematic_path(
                 f"Planning failed! Error code: {res.error_code.val}."
             )
             return None
+    def _plan_cartesian_path(
+        self, wait_for_server_timeout_sec: Optional[float] = 1.0
+    ) -> Optional[JointTrajectory]:
+
+        # Re-use request from move action goal
+        self.__cartesian_path_request.start_state = self.__move_action_goal.request.start_state
+        self.__cartesian_path_request.group_name = self.__move_action_goal.request.group_name
+        self.__cartesian_path_request.link_name = self.__end_effector_name
+        self.__cartesian_path_request.max_step = 0.0025
+
+        stamp = self._node.get_clock().now().to_msg()
+        self.__cartesian_path_request.header.stamp = stamp
+
+        self.__cartesian_path_request.path_constraints = self.__move_action_goal.request.path_constraints
+        for position_constraint in self.__cartesian_path_request.path_constraints.position_constraints:
+            position_constraint.header.stamp = stamp
+        for orientation_constraint in self.__cartesian_path_request.path_constraints.orientation_constraints:
+            orientation_constraint.header.stamp = stamp
+        #for joint_constraint in self.__cartesian_path_request.path_constraints.joint_constraints:
+            #joint_constraint.header.stamp = stamp
```

Comment:
Why is setting of timestamps for joint constraints commented?

Reply:
because the joint constraint has no header for some reason

Reply:
because the joint constraint msg type has no header for some reason, deleting it straight up didnt feel right as it looked like i forgot it... maybe replace it with a text comment ?

Reply:
I think it is good to just remove it then. :smile: